NewFile.txt

```diff
@@ -0,0 +1,7 @@
+This is a new file.
+
+With some new content.
+
+Seriously...
+
+Very important content.
```

Comment:
With reply.

---
In reply to: [68511252](https://github.com/kivancmuslu/GitHubTest/pull/1#discussion_r68511252) [](ancestors = 68511252)